egs/audioset/AT/local/generate_audioset_manifest.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import csv
+import glob
+import logging
+
+import torch
+from lhotse import CutSet, Fbank, FbankConfig, LilcomChunkyWriter
+from lhotse.audio import Recording
+from lhotse.cut import MonoCut
+from lhotse.supervision import SupervisionSegment
+
+from icefall.utils import get_executor
+
+torch.set_num_threads(1)
+torch.set_num_interop_threads(1)
+
+
+def parse_csv(csv_file="downloads/audioset/full_train_asedata_with_duration.csv"):
+
+    mapping = {}
+    with open(csv_file, "r") as fin:
+        reader = csv.reader(fin, delimiter="\t")
+        for i, row in enumerate(reader):
+            if i == 0:
+                continue
+            key = "/".join(row[0].split("/")[-2:])
+            mapping[key] = row[1]
+    return mapping
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+
+    parser.add_argument("--dataset-dir", type=str, default="downloads/audioset")
+
+    parser.add_argument(
+        "--split",
+        type=str,
+        default="balanced",
+        choices=["balanced", "unbalanced", "eval", "eval_all"],
+    )
+
+    parser.add_argument(
+        "--feat-output-dir",
+        type=str,
+        default="data/fbank_audioset",
+    )
+
+    return parser
+
+
+def main():
+    parser = get_parser()
+    args = parser.parse_args()
+
+    dataset_dir = args.dataset_dir
+    split = args.split
+    feat_output_dir = args.feat_output_dir
+
+    num_jobs = 15
```

Comment:
Please follow other scripts to set the `num_jobs`.

Users may not have 15 CPUs on their machines.

Please see also
https://github.com/k2-fsa/icefall/blob/42de4591107868816d240c991b1e8acdfdfd090b/egs/librispeech/ASR/local/compute_fbank_librispeech.py#L81